README.md

```diff
@@ -158,9 +162,109 @@ const friendCounts = new ShardedCounter<Record<Id<"users">, number>>(
 );
 
 // Decrement a user's friend count by 1
-await friendsCount.add(ctx, userId, -1);
+await friendsCount.dec(ctx, userId);
+```
+
+## Reduce contention on reads
+
+Reading the count with `counter.count(ctx, "checkboxes")` reads from all shards
+to get an accurate count. This takes a read dependency on all shard documents.
+
+- In a query subscription, that means any change to the counter causes the query
+  to rerun.
+- In a mutation, that means any modification to the counter causes an
+  [OCC](https://docs.convex.dev/error#1) conflict.
+
+You can reduce contention by estimating the count: read from a smaller number
+of shards and extrapolate based on the total number of shards.
+
+```ts
+const estimatedCheckboxCount = await counter.estimateCount(ctx, "checkboxes");
+```
+
+By default, this reads from a single random shard and multiplies by the total
+number of shards to form an estimate. If the counter was accumulated from many
+small `counter.inc` and `counter.dec` calls, then they should be uniformly
+distributed across the shards, so the estimate will be good.
+
+In some cases the counter will not be evenly distributed:
+
+- If the counter was accumulated from few operations
+- If some operations were `counter.add`s or `counter.subtract`s with large
+  values, because each operation only changes a single shard
+- If the number of shards changed
+
+In these cases, the count might not be evenly distributed across the shards.
+To repair such cases, you can call:
+
+```ts
+await counter.rebalance(ctx, "checkboxes");
 ```
 
+Which will even out the count across shards.
+
+If you reduce the number of shards for a key, you will be left with extra shards
```

Comment:
Todo: make it more explicit that this is if you change options in the ShardedCounter constructor